src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -102,7 +102,7 @@ def initialize(logger)
 
     # Return a two-element array [SchemaId, SchemaString]
     def get_schema_info(record)
-        hashkey = record.keys().sort().join()
+        hashkey = record.keys().join()
```

Comment:
Thanks for catching this. replaced with get_schema_key(); I also removed Log() function and replace it with proper logger.

---
In reply to: [97888784](https://github.com/Azure/fluentd-plugin-mdsd/pull/6#discussion_r97888784) [](ancestors = 97888784)